src/components/radio-button/radio-button-test.stories.tsx

```diff
@@ -147,24 +147,22 @@ const radioContainerWidth = 400;
 export const RadioButtonComponent = (props: Partial<RadioButtonProps>) => {
   const [isChecked, setIsChecked] = React.useState(false);
   return (
-    <>
-      <div
-        style={{
-          marginTop: "64px",
-          marginLeft: "64px",
-          width: radioContainerWidth,
-        }}
-      >
-        <RadioButton
-          id="radio-1"
-          value="radio1"
-          label="Radiobutton 1"
-          checked={isChecked}
-          onChange={(e) => setIsChecked(e.target.checked)}
-          {...props}
-        />
-      </div>
-    </>
+    <div
```

Comment:
**suggestion** (non-blocking): another `div` that could be replaced with a `Box`